### PR TITLE
[FIX] Adding retentionEnabledDefault check before showing warning message 

### LINKED ISSUE #20685: Broken retention policy warning shown in all rooms
### Description:

Retention policy warning appears in all channels, even though no policy is defined for that channel. The warning is also malformed, only showing "30" - not "30 days" etc.

### Steps to reproduce:

1. Enable retention policy
2. Make sure the three "applies to" settings are disabled
3. Apply
4. Check room info for a channel

### Expected behavior:

Room info does not show a warning.

### Actual behavior:

All rooms show a warning:

image

The warning shows an unspecified "30" value (no indication of days, etc.).

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.1
- Operating System: Linux
- Deployment Method: tar
- Number of Running Instances: 1
- NodeJS Version: 12.20.2
- MongoDB Version: 4.2.12

### PR DESCRIPTION
**Pull request by yash-rajpal**
- [x] I have read the Contributing Guide 
  - [x] I have signed the CLA 
  - [x] Lint and unit tests pass locally with my changes
  - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [ ] I have added necessary documentation (if applicable)
  - [ ] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)

Added check for retentionEnabledDefault before showing prune warning message.

## Issue(s)

fixes #20685 
## Steps to test or reproduce

Check issue
## Further comments

Before showing message in RoomInfo, we were only checking for global retentionPolicyEnabled, we were not checking for the channel or dm or group policy. So added check for that.

image
So when these 3 are off, warning should not be shown in respective types of channels.

image

### CODE REVIEW

**Comment by yash-rajpal** [comment]
image

Also fixed sentence a bit, as described in issue, "days" word was missing after number of days

**Comment by yash-rajpal** [comment]
@ggazzo @dougfabris this pr is ready for review. :)

**Comment by yash-rajpal** [comment]
@ggazzo @dougfabris Please have a look. Thanks :)

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -3250,10 +3250,10 @@
   "RetentionPolicy_MaxAge_Groups": "Maximum message age in private groups",
   "RetentionPolicy_Precision": "Timer Precision",
   "RetentionPolicy_Precision_Description": "How often the prune timer should run. Setting this to a more precise value makes channels with fast retention timers work better, but might cost extra processing power on large communities.",
-  "RetentionPolicy_RoomWarning": "Messages older than __time__ are automatically pruned here",
-  "RetentionPolicy_RoomWarning_FilesOnly": "Files older than __time__ are automatically pruned here (messages stay intact)",
-  "RetentionPolicy_RoomWarning_Unpinned": "Unpinned messages older than __time__ are automatically pruned here",
-  "RetentionPolicy_RoomWarning_UnpinnedFilesOnly": "Unpinned files older than __time__ are automatically pruned here (messages stay intact)",
+  "RetentionPolicy_RoomWarning": "Messages older than __time__ days are automatically pruned here",
```
Comment (KevLehman) [inline packages/rocketchat-i18n/i18n/en.i18n.json:3253]: Maybe, to reflect that __time__ can be `1` we can write `day(s)` here 🤔 not sure

**Review by ggazzo — APPROVED** [review]

**Pull request merged**